javascript/ql/src/Security/CWE-020/IncompleteHostnameRegExp.ql

```diff
@@ -46,7 +46,12 @@ where
     or
     exists(Configuration cfg |
       cfg.hasFlow(e.flow(), _) and
-      e.mayHaveStringValue(pattern)
+      e.mayHaveStringValue(pattern) and
+      // ignore strings that are unlikely to be regular expressions for URLs
+      not (
+        pattern.length() > 50 or
```

Comment:
I'm worried that 50 could be a little short, cf. [this regex](https://www.regextester.com/94502) for matching valid URLs (not quite the same thing, I know, but related), which is 63 characters long.